src/components/dataset/details/DatasetDetailView.jsx

```diff
@@ -1,81 +1,105 @@
-import React from 'react';
+import _ from 'lodash';
```

Comment:
Are these linting errors real or spurious?

Reply:
those look legit.  There may be a way to get intellij to run "Organize Imports" on save, which would correct that for you